config/default/kustomization.yaml

```diff
@@ -14,7 +12,7 @@ namePrefix: capi-
 #commonLabels:
 #  someName: someValue
 
-resources:
```

Comment:
@detiber I assume we should undo this change, because `bases` was deprecated in kustomize v2.1.0. Right?

Reply:
If we are trying to align with kubebuilder, then we should probably keep it as `bases`, it *should* still work for the time being...

Reply:
Ok, works for me. We can always change it back 😄